src/main/java/com/alkemy/ong/service/OrganizationService.java

```diff
@@ -5,5 +5,7 @@
 
 public interface OrganizationService {
     OrganizationResponseDto getOrganization(Long id);
+    void updateOrganization(Long id, OrganizationPutDto organization);
+    List<OrganizationValidationErrorDto> validationError(BindingResult result);
```

Comment:
Please validate within the method `updateOrganization`, do not use a public method for this.